Agent workflow fails with multiline messages

## Problem

When triggering an agent workflow with a multiline message, the workflow fails with:

```
[: too many arguments
##[error]Unable to process file command 'output' successfully.
##[error]Invalid format '1. **PR #128 (run-review job)** - Infinite loop issue...
```

## Cause

The workflow uses the message in a shell `[ -z "..." ]` test, and newlines/special characters break the shell parsing.

## Reproduction

```bash
gh workflow run quick --field message="Line 1
Line 2
Line 3"
```

## Fix

Need to properly quote/escape the message variable in the workflow shell script, or use a different approach for the empty check.

--- a/.github/workflows/agent-run.yml
+++ b/.github/workflows/agent-run.yml
@@ -80,11 +80,18 @@ jobs:
 
       - name: Determine message
         id: message
+        env:
+          INPUT_MESSAGE: ${{ inputs.message }}
+          BRANCH_NAME: ${{ steps.branch.outputs.name }}
         run: |
-          if [ -z "${{ inputs.message }}" ]; then
-            echo "msg=Branch: ${{ steps.branch.outputs.name }}" >> $GITHUB_OUTPUT
+          if [ -z "$INPUT_MESSAGE" ]; then
+            echo "msg=Branch: $BRANCH_NAME" >> $GITHUB_OUTPUT
           else
-            echo "msg=${{ inputs.message }}" >> $GITHUB_OUTPUT
+            {
+              echo "msg<<EOF"
+              echo "$INPUT_MESSAGE"
+              echo "EOF"
+            } >> $GITHUB_OUTPUT
           fi
 
       - name: Run CLI

